### ISSUE TITLE
Hardware requirement for training (RTX 5060 8GB)

### ISSUE THREAD

**Issue by UMRme** [Qinkaiyu/CLIP-DR#4]
Hi, I want to use CLIP-DR as my baseline but I'm limited to an RTX 5060 8GB.
With ViT-B/16 and an image size of 224 (or 512), what is the minimum VRAM usage you observed during training?
Also, did you use Gradient Accumulation to fit the model into consumer GPUs? Thanks!
你好，我想用 CLIP-DR 做 Baseline，但我只有 8GB 显存的 RTX 5060。
如果使用 ViT-B/16，图片尺寸设为 224（或 512），请问你们观察到的最低显存占用是多少？
另外，你们是否使用了梯度累加来适配消费级显卡？谢谢！

**Comment by Qinkaiyu** [comment]
Hi, thanks for your interest.
We trained the CLIP-DR baseline on an RTX 4090. I think for ViT-B/16 with 224×224 inputs, batch size 16 would likely require around 11 GB or more during training.

**Comment by UMRme** [comment]
> Hi, thanks for your interest. We trained the CLIP-DR baseline on an RTX 4090. I think for ViT-B/16 with 224×224 inputs, batch size 16 would likely require around 11 GB or more during training.

Thank you so much for your candid and helpful response! I’ll discuss potential hardware resources with my supervisor.
I’ll reach out if I encounter any specific issues while running the baseline. Thanks again for your support and for sharing this great work!
